### DISCUSSION TITLE
Install Rustdesk with commands

### DISCUSSION THREAD

**Discussion by Mylesz3** [rustdesk/rustdesk#5220] [Q&A]
Im trying to make a cmd script that auto install Rustdesk. The problem is that Rustdesk.exe only launch Rustdesk but does not install it, is there any parameter that install Rustdesk in a single command line?

**Comment by tutsde** [comment]
If you start Rustdesk from Rustdesl-1.2.1.x86_64.exe it will unpack files to c:\users\<login>\appdata\local\rustdesk\ and start the rustdesk.exe in this directory.
you can copy this files to c:\program files\ so all user can use it.

**Comment by deep-soft** [comment]
> 

for release version:
rustdesk-1.2.1-x86_64.exe --install

for nightly version:
rustdesk-1.2.2-x86_64.exe --install

**Comment by dinger1986** [comment]
theres a full PowerShell script ready to go here https://rustdesk.com/docs/en/self-host/pro/faq/#can-i-deploy-rustdesk-using-powershell
